### PR TITLE
feat: ✨ add style sd-hidden-links

### PR DESCRIPTION
**Pull request by mariohamann**
## Definition of Reviewable:

- [x] Documentation is created/updated
- [x] Stories (features, a11y) are created/updated
- [ ] relevant tickets are linked

### CODE REVIEW

**Comment by mariohamann** [comment]
@coraliefeil It came a style. Can we align on the documentation upfront here, so we are aligned on dev and design side? :) Especially as my docs currently look... ugly.

**Comment by yoezlem** [comment]
@mariohamann should the copy code section look like this:
image

**Comment by mariohamann** [comment]
Definitely not and I just realized it in this branch is well. Let's do it in a separate ticket! :)

**Comment by christophsaile** [comment]
LGTM, did some screenreader testing and everything looked good :)

**Review by christophsaile — APPROVED** [review]

**Comment by coraliefeil** [comment]
Surrounding content:
Bildschirmfoto 2024-08-14 um 09 25 58

Pls show a single `sd-button`, not `sd-hidden-links--multiple`

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+const { argTypes, parameters } = storybookDefaults('sd-hidden-links');
+const { overrideArgs } = storybookHelpers('sd-hidden-links');
+const { generateTemplate } = storybookTemplate('sd-hidden-links');
+
+// HTML helper to get syntax highlighting and formatting in the template string
+const html = String.raw;
+
+/**
+ * Hidden links can be used to show links only for keyboard users.
```
Comment (coraliefeil) [inline packages/components/src/styles/hidden-links/hidden-links.stories.ts:13]: * Can be used to show links only for keyboard users.

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+export const Default = {
+  args: overrideArgs({
+    type: 'slot',
+    name: 'default',
+    value: html`<sd-button href="#">Skip to content</sd-button>`
+  }),
+  render: (args: any) => {
+    return generateTemplate({
+      options: {
+        templateContent: html`<div class="relative bg-primary-100 w-full h-[256px] p-8">
```
Comment (coraliefeil) [inline packages/components/src/styles/hidden-links/hidden-links.stories.ts:41]: templateContent: html`<div class="relative bg-white w-full h-[256px] p-8">

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+      <p>Tab through this area to see buttons one after another.</p>
+    </div>`
+};
+
+/**
+ * Use the `sd-hidden-links--multiple` modifier to show mutiple `<sd-navigation-item>` elements. This is recommended for >2 links.
+ */
+export const MultipleLinks = {
+  render: () =>
+    html`<div class="relative bg-primary-100 w-full h-[256px] p-8">
```
Comment (coraliefeil) [inline packages/components/src/styles/hidden-links/hidden-links.stories.ts:128]: html`<div class="relative bg-white w-full h-[256px] p-8">

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+    chromatic: {
+      disableSnapshot: true
+    }
+  },
+  args: overrideArgs({ type: 'slot', name: 'default', value: 'Lorem Ipsum' }),
+  argTypes
+};
+
+/**
+ * Per default the hidden links are optimized for a single `<sd-button>`.
```
Comment (coraliefeil) [inline packages/components/src/styles/hidden-links/hidden-links.stories.ts:30]: Delete default section ... that’s already shown at the top of the page.

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+      },
+      args
+    });
+  }
+};
+
+/**
+ * The style appends its position in regard of the parent container's position. It can be used with any element.
+ */
+export const SurroundingContent = {
```
Comment (coraliefeil) [inline packages/components/src/styles/hidden-links/hidden-links.stories.ts:54]: See comment https://github.com/solid-design-system/solid/pull/1261#issuecomment-2288043299

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+            <img class="h-8 md:h-12 lg:h-[56px]" src="images/logo-unioninvestment-lg.svg" alt="Logo" />
+          </a>
+        </div>
+      </sd-header>
+    </div>`;
+  }
+};
+
+/**
+ * You can "stack" multiple `sd-hidden-links` by adding multiple of them to the same parent. This is recommended for 2 links.
```
Comment (coraliefeil) [inline packages/components/src/styles/hidden-links/hidden-links.stories.ts:112]: * "Stack" multiple `sd-hidden-links` by adding multiple of them to the same parent. (Recommended for 2 links)

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+  render: () =>
+    html`<div class="relative bg-primary-100 w-full h-[256px] p-8">
+      <div class="sd-hidden-links"><sd-button href="#">Skip to Search</sd-button></div>
+      <div class="sd-hidden-links"><sd-button href="#">Skip to Content</sd-button></div>
+      <p>Tab through this area to see buttons one after another.</p>
+    </div>`
+};
+
+/**
+ * Use the `sd-hidden-links--multiple` modifier to show mutiple `<sd-navigation-item>` elements. This is recommended for >2 links.
```
Comment (coraliefeil) [inline packages/components/src/styles/hidden-links/hidden-links.stories.ts:124]: * Use the `sd-hidden-links--multiple` modifier to show mutiple `<sd-navigation-item>` elements. (Recommended for >2 links)

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+      <p>Tab through this area to see multiple links.</p>
+    </div>`
+};
+
+/**
+ * Use the `--sd-hidden-links-title` CSS variable to set a title for multiple links. German and English are set by default in regard of the document's or elements `lang` attribute.
+ */
+export const TitleForMultipleLinks = {
+  render: () =>
+    html`<div class="relative bg-primary-100 w-full h-[256px] p-8" lang="de">
```
Comment (coraliefeil) [inline packages/components/src/styles/hidden-links/hidden-links.stories.ts:143]: html`<div class="relative bg-white w-full h-[256px] p-8" lang="de">

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+ * Use the `sd-hidden-links--debug` modifier to always show the links for debugging purposes.
+ */
+export const Debug = {
+  parameters: {
+    chromatic: {
+      disableSnapshot: true
+    }
+  },
+  render: () =>
+    html`<div class="relative bg-primary-100 w-full h-[256px] p-8">
```
Comment (coraliefeil) [inline packages/components/src/styles/hidden-links/hidden-links.stories.ts:184]: html`<div class="relative bg-white w-full h-[256px] p-8">

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts [hunk 10]
```diff
@@ -0,0 +1,192 @@
...
+      </div>
+      <div class="relative bg-neutral-200 w-full h-[256px] p-8">
+        <div class="sd-hidden-links sd-hidden-links--multiple" lang="en">
+          <sd-navigation-item href="#">Search</sd-navigation-item>
+          <sd-navigation-item href="#">Content</sd-navigation-item>
+          <sd-navigation-item href="#">Footer</sd-navigation-item>
+        </div>
+        <p>Here you will see an English title.</p>
+      </div>
+      <div class="relative bg-primary-100 w-full h-[256px] p-8">
```
Comment (coraliefeil) [inline packages/components/src/styles/hidden-links/hidden-links.stories.ts:159]: <div class="relative bg-pwhite w-full h-[256px] p-8">

**Comment by coraliefeil** [comment]
Review done. Pls reassign when adjusted. Thanks a lot.

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts (hunk 4 above)
Comment (mariohamann) [reply packages/components/src/styles/hidden-links/hidden-links.stories.ts:30]: Will be hidden, when the docs PR is merged.

**Pull request merged**